packages/ui/cypress/fixtures/users.ts

```diff
@@ -134,16 +134,35 @@ const users: Record<string, Partial<User> & { groups: UserGroup[] }> = {
     password: hashedPassword,
     groups: [UserGroup.NewUI, UserGroup.GeneralHandler]
   },
-  SupervisorWithExcludedTriggers: {
-    username: "SupervisorWithExcludedTriggers",
-    visibleForces: ["01"],
+  userExcludedTriggers: {
+    username: "userExcludedTriggers",
+    visibleForces: ["007"],
     forenames: "Supervisor1",
-    surname: "WithExcludedTriggers",
-    email: "SupervisorWithExcludedTriggers@example.com",
+    surname: "User",
+    email: "userExcludedTriggers@example.com",
     password: hashedPassword,
     groups: [UserGroup.NewUI, UserGroup.Supervisor],
     excludedTriggers: [TriggerCode.TRPR0001, TriggerCode.TRPR0002, TriggerCode.TRPR0003, TriggerCode.TRPR0008]
   },
+  userExcludedTriggersMultiForces: {
+    username: "userExcludedTriggersMultiForces",
+    visibleForces: ["001", "002", "003"],
+    forenames: "Supervisor2",
+    surname: "User",
+    email: "userExclTriggsMultiForces@example.com",
+    password: hashedPassword,
+    groups: [UserGroup.NewUI, UserGroup.Supervisor],
+    excludedTriggers: [TriggerCode.TRPR0012, TriggerCode.TRPS0008, TriggerCode.TRPR0021, TriggerCode.TRPR0030]
+  },
+  Bichard18: {
+    username: "Bichard18",
+    visibleForces: ["18"],
+    forenames: "Bichard18",
+    surname: "User",
+    email: "bichard18@example.com",
+    password: hashedPassword,
+    groups: [UserGroup.GeneralHandler, UserGroup.NewUI]
+  },
```

Comment:
The numberedUsers() function generates users with access to visible forces up to 5. We needed to assign a force to the user that has no triggers excluded at the force level, allowing us to display all trigger codes on the filter panel. Since Force 18 is the only one without excluded triggers, I created a user with Force 18 as their visible force.

Reply:
Ahhhh okay, makes sense - maybe we can rename it so it says it's like `userWithoutExcludedTriggers` or something then so it's clearer than `Bichard18`?